mako.5.scd

```diff
@@ -298,6 +297,15 @@ associated command-line option.
 
 	Default: 0
 
+*surface* = _surface_
```

Comment:
This is a leftover from previous code since the `surface` option has been removed, correct?